packages/react/src/components/toggle-switch/toggle-switch.test.tsx

```diff
@@ -23,6 +23,28 @@ describe('ToggleSwitch', () => {
         expect(callback).toHaveBeenCalledTimes(0);
     });
 
+    test('renders label to the right when labelPosition is "right" (by default)', () => {
+        const { container } = renderWithProviders(
+            <ToggleSwitch label="Switch" toggled onToggle={jest.fn()} />,
+        );
+
+        const label = container.querySelector('label');
+        const button = container.querySelector('button');
+
+        expect(button?.nextSibling).toBe(label);
+    });
```

Comment:
Je rajouterais peut-être plutôt un testid sur le label, et j'utiliserais le `getByRole` pour la switch.
```suggestion
    test('renders label to the right when labelPosition is "right" (by default)', () => {
        const { getByTestId, getByRole } = renderWithProviders(
            <ToggleSwitch label="Switch" toggled onToggle={jest.fn()} />,
        );

        const label = getByTestId('switch-label');
        const button = getByRole('switch');

        expect(button?.nextSibling).toBe(label);
    });
```